packages/realm/scripts/build-android.ts

```diff
@@ -15,19 +15,25 @@
 // limitations under the License.
 //
 ////////////////////////////////////////////////////////////////////////////
+/* eslint-disable no-console */
 
-const commandLineArgs = require("command-line-args");
-const fs = require("fs-extra");
-const path = require("path");
-const exec = require("child_process").execFileSync;
-
-const { version } = require("realm/package.json");
+import commandLineArgs from "command-line-args";
+import fs from "node:fs";
+import path from "node:path";
+import { execFileSync as exec } from "node:child_process";
+import { version } from "../package.json";
 
 const packageRoot = path.resolve(__dirname, "..");
 
 const NDK_VERSION = "25.1.8937393";
 
 const { ANDROID_SDK_ROOT } = process.env;
+
+if (!ANDROID_SDK_ROOT) {
+  console.error(`Missing env variable ANDROID_SDK_ROOT`);
+  process.exit(1);
+}
+
```

Comment:
Now I understand why you wanted to hold off the merge of #6402 😬

Reply:
Yeah, but it should be easily merged.